configs/m1_firestorm-sst.yaml

```diff
@@ -0,0 +1,307 @@
+# M1 Firestorm core
```

Comment:
Would it be better to put all sst related config files in the sst folder itself?

Reply:
Resolving as it's a duplicate comment that just bloats the PR.